src/components/pages/NewProject/components/MemberList.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Dropdown, Form } from "react-bootstrap";
+
+// API get route
+import projectService from "../../../../Services/ProjectService";
+
+function MemberList(props) {
+  const teamMember = projectService.getAllUser;
```

Comment:
You haven't used the getAllUser method here - you have just assigned it to a new variable.
It needs to be wrapped in an async function and called using:
`const teamMember = await projectService.getAllUser();`

Reply:
thx for reminding, haven't tried that before.

Reply:
update completed